releasenotes/notes/0.3/0.3-summary-8bdd0273e78a9795.yaml

```diff
@@ -0,0 +1,116 @@
+---
+prelude: >
+    This release has several major changes:
+      - Adds perturbation theory sub module for computing and utilizing the Dyson series
+        and Magnus expansion in a multi-variable setting.
```

Comment:
Should this say Dyson and Magnus exp of Hamiltonian and Lindblad models?

Reply:
Added a line saying it can be applied to both Hamiltonian and Lindblad dynamics, and more generally any LMDE.